dataactbroker/permissions.py

```diff
@@ -16,7 +17,16 @@
 
 
 def requires_login(func):
-    """Decorator requiring that _a_ user be logged in (i.e. that we're not using an anonymous session)"""
+    """ Decorator requiring that a user be logged in (i.e. that we're not using an anonymous session)
+
+        Args:
```

Comment:
I think you wrote this in the wrong place, none of these is an arg here and the return is incorrect.